### DISCUSSION TITLE
Not require the websites I visit for CUDA on WSL2

### DISCUSSION THREAD

**Discussion by lfuelling** [microsoft/WSL#6559] [General]
**Is your feature request related to a problem? Please describe.**
I have wasted the last seven hours trying to get CUDA to work in WSL2.
It seems like I have to enable "optional diagnostics" (ie. the websites I visit and lots of more personal stuff that shouldn't concern you **at all**) just to have the `/usr/lib/wsl` folder mounted which seemingly is the only thing keeping CUDA from working.

**Describe the solution you'd like**
I'd like to keep my personal data to myself and still being able to use CUDA on WSL2.

**Describe alternatives you've considered**
Remove the privacy unfriendly OS, and install Linux directly.

**Comment by therealkenc** [comment]
CUDA is only available on Insiders by-design (20150 or better which is [documented](https://docs.microsoft.com/en-us/windows/win32/direct3d12/gpu-cuda-in-wsl)). There is no actionable here except to wait for the general release. Which is 21H1 AFAIK.

**Reply by lfuelling** [reply]
Yes I read that. 
Why does Microsoft want my visited websites (amongst other very personal data) for this?
Should I also write a diary and address it to all-employees@microsoft.com?

**Reply by therealkenc** [reply]
Information on the windows Insider program is [here](https://docs.microsoft.com/en-us/answers/products/). Windows Insider community forum is [here](https://answers.microsoft.com/en-us/insider/forum?sort=LastReplyDate&dir=Desc&tab=All&status=all&mod=&modAge=&advFil=&postedAfter=&postedBefore=&threadType=All&isFilterExpanded=false&page=1).
